[ERROR] Error running: python3 /home/kingusmao/tcc/blocks-world/fd/fast-downward.py /home/kingusmao/tcc/blocks-world/domain.pddl /home/kingusmao/tcc/blocks-world/problem.pddl --landmarks "lm=lm_exhaust(reasonable_orders=false, only_causal_landmarks=false, disjunctive_landmarks=false, conjunctive_landmarks=true, no_orders=false)" --heuristic "hlm=lmcount(lm)" --search "astar(lmcut())" > /home/kingusmao/tcc/blocks-world/output.txt
